### PR TITLE
update operator doc and add me back to reviewer

### PR DESCRIPTION
**Pull request by adrian555**
What is in this PR:
- adds the current tested operator image for each kfctl release
- adds operator reconciler handling of events
- revises some operator document wording
- adds myself back to reviewer so to receive review requests

### CODE REVIEW

**Comment by adrian555** [comment]
cc @animeshsingh

File: operator.md [hunk 1]
```diff
@@ -126,27 +149,48 @@ kubectl delete mutatingwebhookconfigurations katib-mutating-webhook-config
...
 make push-operator
 ```
+
+> Note: replace **<docker_repo>** with the image repo name and tag, for example, `docker.io/example/kubeflow-operator:latest`.
+
+3. Follow [Deployment Instructions](#deployment-instructions) section to test the operator with the newly built image
+
+## Current Tested Operators and Pre-built Images
+
+Kubeflow Operator controller logic is based on the [`kfctl` package](https://github.com/kubeflow/kfctl/tree/master/pkg), so for each release of `kfctl`, an operator image is built and tested with that version of [`manifests`](github.com/kubeflow/manifests) to deploy a _KfDef_ instance. Following table shows what have been tested.
```
Comment (animeshsingh) [inline operator.md:187]:
```suggestion
Kubeflow Operator controller logic is based on the [`kfctl` package](https://github.com/kubeflow/kfctl/tree/master/pkg), so for each major release of `kfctl`, an operator image is built and tested with that version of [`manifests`](github.com/kubeflow/manifests) to deploy a _KfDef_ instance. Following table shows what releases have been tested.
```

**Pull request merged**